integration/raft/channel_participation_test.go

```diff
@@ -660,20 +660,112 @@ var _ = Describe("ChannelParticipation", func() {
 				Height:            3,
 			})
 
-			By("submitting a channel config update for the system channel")
+			By("creating a second channel that will have three consenters")
+			network.CreateChannel("testchannel2", orderer1, org1peer0)
+
+			expectedChannelInfo = channelparticipation.ChannelInfo{
+				Name:              "testchannel2",
+				URL:               "/participation/v1/channels/testchannel2",
+				Status:            "active",
+				ConsensusRelation: "consenter",
+				Height:            1,
+			}
+			for _, o := range orderers1and2 {
+				By("listing single channel for " + o.Name)
+				Eventually(func() channelparticipation.ChannelInfo {
+					return channelparticipation.ListOne(network, o, "testchannel2")
+				}, network.EventuallyTimeout).Should(Equal(expectedChannelInfo))
+			}
+
+			for _, o := range orderers1and2 {
+				By("listing the channels for " + o.Name)
+				cl := channelparticipation.List(network, o)
+				channelparticipation.ChannelListMatcher(cl, []string{"testchannel", "testchannel2"}, []string{"systemchannel"}...)
```

Comment:
```suggestion
				channelparticipation.ChannelListMatcher(cl, []string{"testchannel", "testchannel2"}, "systemchannel")
```
A nit, but one thing I'd like to see disappear from these tests. :) I fixed all the existing places in another PR but let's not add new ones.

Reply:
done